package.json

```diff
@@ -74,7 +74,8 @@
     "**/d3-color": "^3.1.0",
     "**/@miksu/prettier/minimatch": "^3.0.8",
     "**/request/qs": "^6.11.0",
-    "nodegit/ramda": "^0.27.1"
+    "nodegit/ramda": "^0.27.1",
+    "**/make-fetch-happen/http-cache-semantics": "^4.0.0"
```

Comment:
Why not "^4.1.1"? It resolves the same, but it's more clear to actually list the version with the fix.

Reply:
+1

Reply:
```
yarn why node-gyp
yarn why v1.22.19
[1/4] Why do we have the module "node-gyp"...?
[2/4] Initialising dependency graph...
warning Resolution field "trim@0.0.3" is incompatible with requested version "trim@0.0.1"
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "node-gyp@8.4.1"
info Has been hoisted to "node-gyp"
info Reasons this module exists
   - Hoisted from "node-sass#node-gyp"
   - Hoisted from "codesandbox#pacote#@npmcli#run-script#node-gyp"
info Disk size without dependencies: "3.25MB"
info Disk size with unique dependencies: "4.23MB"
info Disk size with transitive dependencies: "8.56MB"
info Number of shared dependencies: 82
=> Found "nodegit#node-gyp@4.0.0"
info This module exists because "nodegit" depends on it.
info Disk size without dependencies: "1.92MB"
info Disk size with unique dependencies: "3.28MB"
info Disk size with transitive dependencies: "14.61MB"
info Number of shared dependencies: 80
Done in 0.84s.
```

Reply:
* nodegit is at version 0.27.0 which is the latest public version https://github.com/nodegit/nodegit/releases/tag/v0.27.0

* make-fetch-happen bumps http-cache-semantics to 4.1.1 at @11.0.3. but in the latest node-gyp it hasn't picked up the change yet  [here](https://github.com/nodejs/node-gyp/blob/39ac2c135db8a9e62bf22f0c7a4469ae6c381325/package.json#L28).

Therefore, we could only resolve it.

Reply:
@BSFishy could you either resolve do 
```
"**/make-fetch-happen": "^11.0.3" 
```
or 
```
"**/make-fetch-happen/http-cache-semantics": "^4.1.1"
```

Reply:
Updated resolution :+1: